### DISCUSSION TITLE
[Bug] 主页动态显示为“鉴权失败，请联系账号组”

### DISCUSSION THREAD

**Discussion by SuanOrange** [ywmoyue/biliuwp-lite#575] [bug]
### Bug 描述

扫码登陆后在主页界面的动态页无内容，显示“鉴权失败，请联系账户组”

### 操作步骤

```plain text
1.打开应用(版本为v4.6.29.0)
2.在主页界面点击“动态”，显示“鉴权失败，请联系账号组”
```

### 期望结果

正常显示动态内容

### 截图

![Screenshot 2024-04-11 184358](https://github.com/ywmoyue/biliuwp-lite/assets/67503405/0d53a294-a049-4c5e-a2df-6f6430da19b5)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 10 22H2 (Build 19045)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 备注

有概率会显示“追番追剧”内容，但是有概率在其他版块中会显示，并且在哪一个板块中都显示不全，大概率显示为“鉴权失败，请联系账号组”
![在专栏区显示](https://github.com/ywmoyue/biliuwp-lite/assets/67503405/80f3f158-885f-423f-8c62-313514d8b44f)

### 代码贡献

我更擅长发现问题

**Comment by ShellShield** [comment]
和你一样的情况，昨天还好好的，我还以为我是个例
